### PR TITLE
Remove p in favor of pl, py and pr

### PR DESCRIPTION
**Pull request by arzezak**
On Infirmary, sortable tables are showing a misaligned handle/position. I believe this is being caused by Tailwind's `p` directive overriding the `pl` that causes the position to be moved to the right of the handle.

Screenshot 2023-08-31 at 5 34 14 PM

Screenshot 2023-09-01 at 9 15 56 AM

### CODE REVIEW

**Comment by arzezak** [comment]
Another option besides removing it would be to assign `py-3` (this probably makes more sense).

Any thoughts @tysongach?

**Comment by Simonpedro** [comment]
As an alternative approach, we could put the svg inline (as we usually do) inside a new cell instead of using `background: url(...)`, although would require more changes. I remember talking with @tute about this, not sure if he has any additional thoughts.

**Review by tute — APPROVED** [review]
I don't have an opinion on what's the better alternative here. This LGTM FWIW.

**Comment by tute** [comment]
@arzezak can you please post a screenshot of the "after" change?

**Comment by arzezak** [comment]
> @arzezak can you please post a screenshot of the "after" change?

Line 1 of the screenshot has the proposed change applied, @tute.

**Review by Simonpedro — APPROVED** [review]

File: app/components/anchor/table_component.html.erb [hunk 1]
```diff
@@ -25,7 +25,7 @@
             maybe_link(model, column, index, self).to_s.html_safe,
             class: class_names(
               "relative",
-              "p-3 pl-8 bg-[url('assets/icons/menu.svg')] bg-[length:16px_16px] bg-[8px_center] bg-no-repeat": sortable? && index.zero?,
+              "pl-8 bg-[url('assets/icons/menu.svg')] bg-[length:16px_16px] bg-[8px_center] bg-no-repeat": sortable? && index.zero?,
```
Comment (tysongach) [inline app/components/anchor/table_component.html.erb:28]:
```suggestion
              "pl-8 py-3 pr-2 bg-[url('assets/icons/menu.svg')] bg-[length:16px_16px] bg-[8px_center] bg-no-repeat": sortable? && index.zero?,
```

**Review by tysongach — COMMENTED** [review]
> I believe this is being caused by Tailwind's `p` directive overriding the `pl`

Yeah, I have never understood why they put the `padding` shorthand below the `padding-[side]`s in the source order. Feels backwards to me…

I’d recommend spelling out each side. If we _only_ set the left padding, it’s odd to me that this cell has no top, bottom, or right padding, but other cells do:

Screenshot 2023-09-01 at 07 02 20

File: app/components/anchor/table_component.html.erb (hunk 1 above)
Comment (arzezak) [reply app/components/anchor/table_component.html.erb:28]: 👍 Sounds good.

**Pull request merged**